django-verdant/rca/static/rca/css/streamfield-blocks/news.less

```diff
@@ -0,0 +1,206 @@
+.news-block {
+
+    &__header {
+        @media @media-desktop-small {
+            display: flex;
+            align-items: center;
+            justify-content: space-between;
+        }
+    }
+
+    &__title {
+        margin-bottom: @spacing-two;
+    }
+
+    &__more {
+        display: none;
+
+        @media @media-desktop-small {
+            display: flex;
+            align-items: center;
+            font-size: 14px;
+            text-transform: uppercase;
+            color: @color-highlight-blue;
+            font-family: @font-family-benton;
+        }
+
+        &:hover {
+            color: @color-pink;
+        }
+    }
+
+    &__arrow {
+        width: 39px;
+        height: 30px;
+        margin-left: @spacing-one;
+        display: inline-block;
+        fill: @color-highlight-blue;
+
+        .news-block__more:hover & {
+            fill: @color-pink;
+        }
+    }
+
+    &__wrapper {
+        @media @media-desktop-small {
+            display: grid;
+            display: -ms-grid;
+            grid-template-columns: 2fr 3fr 2fr;
+            -ms-grid-columns: 2fr 3fr 2fr;
```

Comment:
small thing but would you not want to put the browser specific lines first so they act as a fallback, I'm not sure it matters in the long run, just seems odd to see them this way around